README.md

```diff
@@ -4,7 +4,7 @@ SourceKit-LSP is an implementation of the [Language Server Protocol](https://mic
 
 ## Getting Started
 
-SourceKit-LSP is under heavy development! The best way to try it out is to install a recent Swift development toolchain from https://swift.org/download/#snapshots. Make sure you build your package with the same toolchain as you got sourcekit-lsp from to ensure compatibility.
+SourceKit-LSP is under heavy development! The best way to try it out is to install a recent Swift development toolchain from https://swift.org/download/#snapshots. Make sure you build your package with the same toolchain as you got sourcekit-lsp from to ensure compatibility. If you're looking for a Swift client to this LSP process you could consider [https://github.com/ChimeHQ/SwiftLSPClient](https://github.com/ChimeHQ/SwiftLSPClient)
```

Comment:
Please remove this unrelated change. This is also not a good place to advertise such a library.

Reply:
Was just a thought. It seems to be the best option out there.